Source/Public/ConvertTo-Bicep.ps1

```diff
@@ -21,22 +21,44 @@
     Go to module repository https://github.com/StefanIvemo/BicepPowerShell for detailed info, reporting issues and to submit contributions.
 #>
 function ConvertTo-Bicep {
+    [CmdletBinding()]
     param(
-        [string]$Path = $pwd.path
+        [string]$Path = $pwd.path,
+        
+        [string]$OutputDirectory = $pwd.path,
```

Comment:
Same as for `Build-Bicep` should not use `$pwd.path` as default OutputDirectory.